woocommerce/woocommerce-gateway-stripe
# Fix E2E tests in latest WC

Fixes: #2892

## Changes proposed in this Pull Request:

This PR includes the minimum changes needed to fix the E2E tests.

- update setup to create shortcode versions of the cart and the checkout (Woo creates blocks cart and checkout pages by default now)
- update the credit card error message selector to use the new CSS classes used for the checkout messages (but keep the old CSS classes for inline messages below the credit card number)
- update the iframe name of the Stripe 3DS modal and the name of the continue button
- update renewal flow to use blocks checkout for `Renew order`

## Testing instructions

1. Create a new Jurassic Ninja site.
2. Update the variables in your `./tests/e2e/config/local.env` file with the new site info.
     - Check the file `./tests/e2e/config/local.env.example` for default values (ADMIN_USER, SSH_HOST and SSH_PATH)
4. Run the E2E setup
    - `npm run test:e2e-setup -- --base_url=<JN_SITE_URL>`
5. Check that all tests pass:
    ![Screenshot 2024-02-14 at 17 19 10](https://github.com/woocommerce/woocommerce-gateway-stripe/assets/407542/4401de57-146a-495d-86d4-2ec1a04b49e8)
